sql/plan.go

```diff
@@ -136,6 +136,9 @@ type planNode interface {
 	Next() bool
 	// Err returns the error, if any, encountered during iteration.
 	Err() error
+	// ExplainPlan fills in name and description for the node and returns a list
+	// of child nodes.
+	ExplainPlan(name, description *string) []planNode
```

Comment:
Switched to return values. Doesn't really affect the code much at all, but I suppose it is more idiomatic.